test/db/io/srec

```diff
@@ -0,0 +1,7 @@
+NAME=srec_test1.txt
+FILE=bins/srec/srec_test1.txt
```

Comment:
seems that you wrote only a test to read, but not to write. @XVilka what do you think? should we have a written one also?

Reply:
Yes, since it's IO plugin, having a write test is important. Btw, it would be nice to add write test to the `ihex://` too: `test/db/io/ihex`.